Tasks/ShellScript/task.json

```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
+            "type": "boolean",
+            "label": "Disable Auto CWD",
+            "defaultValue": "false",
```

Comment:
Consider tying the visibility of the cwd input to this flag. If the checkbox is not checked (default behavior) the working directory input shouldn't be visible at all.